libs/books/feature/src/lib/book-search/book-search.component.html

```diff
@@ -6,7 +6,8 @@
       type="search"
       placeholder="Search for books to add to your reading list"
       aria-label="Search for books to add to your reading list"
-      formControlName="term"/>
+      formControlName="term"
+      (keyup)="onSearchTermKeyUp()"/>
```

Comment:
This is a really messy way to do this. Isn't there somewhere we can get a list of the `valueChanges` for this form control?

Reply:
I agree. I made relevant changes